www/src/Example.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { Fragment } from 'react';
 import rehypeSanitize, { defaultSchema } from 'rehype-sanitize';
+import rehypePrettyCode from 'rehype-pretty-code';
```

Comment:
`rehype-pretty-code` dependency is not included in `package.json`. @vishwamartur